### PR TITLE
chore(rich-text-input): simplify testing

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

Simplifies testing by not dealing directly with JSON values, and instead setting the values with HTML.

Also adds a test with some "unknown" tags, (img and anchor tag), to show that our Rich Text doesn't crash when it encounters HTML it doesn't "know"

### CODE REVIEW

**Comment by montezume** [comment]
There should be some VRT diff due to using different "large value"

File: src/components/inputs/rich-text-input/rich-text-input.spec.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+  it('should call onFocus when focused', () => {
+    const onFocus = jest.fn();
+    const { container } = render(
+      <RichTextInput {...baseProps} onFocus={onFocus} id="foo-bar" />
+    );
+    const input = container.querySelector('#foo-bar');
+    input.focus();
+    expect(onFocus).toHaveBeenCalled();
+    // expect(container.querySelector("[id='foo-bar']")).toBeInTheDocument();
```
Comment (tdeekens) [inline src/components/inputs/rich-text-input/rich-text-input.spec.js]: ?

File: src/components/inputs/rich-text-input/rich-text-input.spec.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+        const button = getByText('Expand');
+        expect(button).toBeInTheDocument(); // is this line necessary since getByText will fail if it's not found :thinking:
+        button.click();
+        expect(getByText('Collapse')).toBeInTheDocument();
+      });
+    });
+  });
+  describe('when defaultExpandMultilineText is disabled', () => {
+    describe('when height of text is less than 32', () => {
+      const originalOffsetHeight = Object.getOwnPropertyDescriptor(
```
Comment (tdeekens) [inline src/components/inputs/rich-text-input/rich-text-input.spec.js]: Do you really have to restore this? Isn't each spec in it's on process anyway?

File: src/components/inputs/rich-text-input/rich-text-input.spec.js (hunk 1 above)
Comment (montezume) [reply src/components/inputs/rich-text-input/rich-text-input.spec.js]: oh, made a mistake in last commit.

**Pull request merged**